baselines/constraint/constraint.py

```diff
@@ -25,6 +100,29 @@ def __init__(self,
         self.a_active = a_active
 
     def step(self, obs, action, done):
+        """
+            Take a step in the constraint mechanism.
+
+            Runs one step of the constraint mechanism
+            which takes steps in paralell with the MDP
+            it is attached to.
+        
+            Parameters
+            ----------
+            #TODO: Requesting explanation of paramters
```

Comment:
These are the observation (before augmentation), the action that was taken, and whether the episode is done